products-page-generator/go.mod

```diff
@@ -1,20 +1,23 @@
 module github.com/opencontainers/oci-conformance/products-page-generator
 
-go 1.17
+go 1.22.6
+
+// See https://github.com/darccio/mergo#important-notes
+replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.16
```

Comment:
This seems like a sign we'd want to either update our code or swap out the dependency.